llarp/linux/service_manager.cpp

```diff
@@ -0,0 +1,61 @@
+#include <llarp/util/service_manager.hpp>
+
+#if defined(WITH_SYSTEMD)
+#include <systemd/sd-daemon.h>
+#include <cassert>
+#endif
```

Comment:
I feel like this should get split into two separate files: A no-op version, and a systemd version and we conditionally compile base on whether we have systemd support.